### PR TITLE
Update ww faqs post pdx conf

### PR DESCRIPTION
**Pull request by canncrochet**
Add new FAQs to ww guide. Add press play to watch stage talks.

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]
LGTM

File: docs/conf/portland/2021/welcome-wagon.rst [hunk 1]
```diff
@@ -337,4 +359,4 @@ What can I do at the conference?
 Where can I learn more?
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-- Read the `_Welcome Wagon guide <https://www.writethedocs.org/conf/portland/2021/welcome-wagon/>`__ to the conference, the `_Unconference Cheat Sheet <https://www.writethedocs.org/conf/portland/2021/unconference-cheatsheet/>`__, `_the Writing Day Cheat Sheet <https://www.writethedocs.org/conf/portland/2021/writing-day-cheatsheet/>`__, and watch the `_virtual conference intro video <https://youtu.be/aLtnc0ITzok>`__. You can learn how to reach out to attendees, what happens at each event, and how to get more help.
+- Read the `Welcome Wagon guide <https://www.writethedocs.org/conf/portland/2021/welcome-wagon/>`__ to the conference, the `Unconference Cheat Sheet <https://www.writethedocs.org/conf/portland/2021/unconference-cheatsheet/>`__, `the Writing Day Cheat Sheet <https://www.writethedocs.org/conf/portland/2021/writing-day-cheatsheet/>`__, and watch the `virtual conference intro video <https://youtu.be/aLtnc0ITzok>`__. You can learn how to reach out to attendees, what happens at each event, and how to get more help.
```
Comment (plaindocs) [inline docs/conf/portland/2021/welcome-wagon.rst]:
```suggestion
Read the `Welcome Wagon guide <https://www.writethedocs.org/conf/portland/2021/welcome-wagon/>`__ to the conference, the `Unconference Cheat Sheet <https://www.writethedocs.org/conf/portland/2021/unconference-cheatsheet/>`__, `the Writing Day Cheat Sheet <https://www.writethedocs.org/conf/portland/2021/writing-day-cheatsheet/>`__, and watch the `virtual conference intro video <https://youtu.be/aLtnc0ITzok>`__. You can learn how to reach out to attendees, what happens at each event, and how to get more help.
```

**Comment by plaindocs** [comment]
Not sure why it thinks there is a list issue on 343 at a casual glance. I'll check the branch out and take a look when I have a moment.

**Comment by plaindocs** [comment]
ah, that one should fix it 76b72f9 -> I didn't spot that when I looked. :crossed_fingers:

**Comment by mxsasha** [comment]
> ah, that one should fix it [76b72f9](https://github.com/writethedocs/www/commit/76b72f9fc705be194aea97c5ca81ba176cce7e97) -> I didn't spot that when I looked. 🤞

Yeah the build works now, I'm just not sure if the layout is right too. Multi-layer lists always confuse me in RST.

**Comment by mxsasha** [comment]
Looks good on https://writethedocs-www--1522.org.readthedocs.build/conf/portland/2021/welcome-wagon/

We should try to extract this content from Portland 2021 and re-use it, but I can take care of that - I made a trello card.

**Pull request merged**